mc-oblivious-ram/src/evictor/mod.rs

```diff
@@ -278,4 +566,134 @@ mod internal_tests {
         let test_branch = deterministic_get_next_branch_to_evict(3, 8);
         assert_eq!(test_branch, 8);
     }
+    #[test]
+    /// Compare prepare deepest with non oblivious prepare deepest and
+    /// prepare_target with non oblivious prepare target
+    fn test_prepare_deepest_and_target() {
```

Comment:
it would be great if we could record a small random tree, and run prepare deepest and prepare target and assert that they have specific results. then someone could compare this with the paper and see that we get the expected result